metadata-service/plugin/src/test/resources/plugin-jar-from-jarFileName/config.yml

```diff
@@ -0,0 +1,20 @@
+plugins:
+  - name: "apache-ranger-authorizer"
+    type: "authorizer"
+    enabled: "true"
+    params:
+      className: "com.datahub.authorization.ranger.RangerAuthorizer"
+      jarFileName: "apache-ranger-authorizer-v2.0.1.jar"
+      configs:
+        username: "foo"
+        password: "bar"
+
+  - name: "apache-ranger-authenticator"  # duplicate name
```

Comment:
Not a duplicate name?

Reply:
It is because of copy paste :),  I deleted this comment